### PR TITLE
Refactoring Music.svelte to use standard tunes.

### PR DESCRIPTION
**Pull request by irshadshalu**
The calculations done for this change is detailed in #12 . And this PR will unblock #12 . 

@wcarss , Do check and review :)

### CODE REVIEW

File: src/Music.svelte [hunk 1]
```diff
@@ -3,24 +3,12 @@
...
-	let noteValues = [
-		246.94, 277.18, 311.13, 369.99,
-		392.00, 415.30, 466.16, 554.37,
-		587.33, 622.25, 659.26, 739.99,
-		830.61, 932.33, 987.77, 1046.50,
-		1108.73, 1244.51, 1479.98, 1661.22,
-		1864.66, 2093.00, 2217.46, 2637.02,
-		2959.96, 3322.44, 3520.00, 3729.31
+	let notes = [
+		'B4', 'C#4', 'F#4', 'G#4',
```
Comment (wcarss) [inline src/Music.svelte:7]: mistakenly commented on the issue -- this should be 'B3' at the start, I think

**Comment by wcarss** [comment]
aside from that, looks good to me 👍

File: src/Music.svelte (hunk 1 above)
Comment (irshadshalu) [reply src/Music.svelte:7]: Oh yes, I missed out that one because I added the 4 later. How silly of me. Will fix it

**Comment by wcarss** [comment]
👍

**Pull request merged**